<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formsale700</class>
 <widget class="QWidget" name="Formsale700">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Formsale700{border-image: url(:/image/sale.png);}</string>
  </property>
  <widget class="QLabel" name="label_F4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>5</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>F4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_stable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>----</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_weight">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿微米黑</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>000000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_mode">
   <property name="geometry">
    <rect>
     <x>409</x>
     <y>4</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;AR PL UKai CN&quot;;</string>
   </property>
   <property name="text">
    <string>----</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_wifi">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>5</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/wlant.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ACDC">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>3</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/dc2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_plu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>43</y>
     <width>791</width>
     <height>71</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 36pt &quot;文泉驿正黑&quot;;</string>
   </property>
   <property name="text">
    <string>--------------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tare">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>000000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_price">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿微米黑</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>000000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_zero">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>----</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿微米黑</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>000000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_F2">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>5</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>F2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_describe">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>435</y>
     <width>791</width>
     <height>33</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;文泉驿正黑&quot;;</string>
   </property>
   <property name="text">
    <string>--------------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_QTime">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>-1</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>2020-01-01 12:30:30</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_F3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>5</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>F3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_F1">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>5</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>F1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_net">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>----</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ver">
   <property name="geometry">
    <rect>
     <x>462</x>
     <y>6</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>----</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>467</x>
     <y>45</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>商户:测试商户</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cas">
   <property name="geometry">
    <rect>
     <x>467</x>
     <y>87</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>市场信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Externalweb">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>5</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/On-line-wai.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_update">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>371</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
